ra2ce/analyses/analysis_config_data/analysis_config_data.py

```diff
@@ -88,11 +88,14 @@ class AnalysisSectionIndirect(AnalysisSectionBase):
     duration_event: float = math.nan
     duration_disruption: float = math.nan
     fraction_detour: float = math.nan
-    fraction_drivethrough: float = math.nan
+    fraction_drivethrough: float = 0
     rest_capacity: float = math.nan
     maximum_jam: float = math.nan
-    partofday: str = ""
-    # accessiblity analyses
+    partofday: str = "day"  # "day, "evening"
```

Comment:
Is there a reason why this is `partofday` instead of `part_of_day` ? This would fit better as an `Enum`.

Reply:
Created an Enum. Shall we have a talk about the Enum way of working? I am curious